README.md

```diff
@@ -17,6 +17,47 @@ You have two options for installation:
 - Copy "eldes_alarm" folder to the "/config/custom_components" folder.
 - Restart HA server.
 
+## Usage
+
+### Events
+
+#### Event attributes
+
+- `alarms` -> holds events with alarms
+- `user_actions` -> holds events with user action like arm and disarm
+- `events` -> holds all other events
+
+#### Display events
+
+To display events used another integration:
+https://github.com/gadgetchnnel/lovelace-home-feed-card
+
+#### Example config for lovelace-home-feed-card
```

Comment:
Add a link to `lovelace-home-feed-card` integration if possible.

Reply:
Added above, but will try to add as hyperlink